src/documents/tutorial-userInterfaceOptions/UserInterfaceOptions.md

```diff
@@ -71,7 +133,9 @@ Insert the following markup at the beginning of your `<body>` tag:
         <!-- This div is for the sliding panel that shows and hides the Preference Editor controls -->
         <div class="fl-panelBar">
             <span class="fl-prefsEditor-buttons">
-                <button id="reset" class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
+                <button id="reset" class="flc-prefsEditor-reset fl-prefsEditor-reset">
+                    <span class="fl-icon-undo"></span>Reset
+                </button>
```

Comment:
Well this is also problematic now because the v2.0 markup and in development one are different. Now we have two panel bars, one for mobile and one for desktop views. We might want to decide what we are documenting in this file exactly. Is it latest, or last stable? If the latter, we may need to change the NPM install instructions or heading.

Reply:
Would it be sufficient if we document the stable version, and adding a note about the differences in the latest and linking to another relevant resource?

Reply:
I'm actually thinking about this again and I believe we've been documenting what's in master in the general docs. In the future we want to indicate version numbers for changes. What we could/should also include is migration information which we have examples of in the docs.

Reply:
Ok. I'll update the doc to reflect what is in master, and add a separate bit of documentation to cover 2.0.